src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
Linking directly to an SCSS file is not valid. You need to compile the SCSS into a CSS file and link to the resulting CSS file instead.